server_admin/topics/users/viewing.adoc

```diff
@@ -1,24 +1,22 @@
-[searching-users]
+[id="proc-searching-users_{context}"]
 = Searching for users
 
-You can search for a user in the database, by using the full name, last name, or email address of the user.
+Search for users to view their detailed information and to understand their role and responsibilities in the console. You can search for a user in the database using the full name, last name, first name, or email address of the user.
```

Comment:
The main goal would be to see how the user is configured and their groups and roles.  The assigned groups and roles determine what application rights they have.  
So I suggest "Search for users to view detailed information about them and their groups and roles."